compact_test.go

```diff
@@ -653,7 +653,8 @@ func TestCompaction_populateBlock(t *testing.T) {
 			expErr:         errors.New("found chunk with minTime: 10 maxTime: 20 outside of compacted minTime: 0 maxTime: 10"),
 		},
 		{
-			// No special deduplication expected.
+			// Deduplication expected.
+			// Introduced by https://github.com/prometheus/tsdb/pull/539.
```

Comment:
Introduced by 370 and 539, as merging chunks happened in 370 and 539 is a bug fix in that.